src/data/productDetails/productDetails.js

```diff
@@ -0,0 +1,48 @@
+import image1 from "./ProductImages/image1.png";
+import image2 from "./ProductImages/image2.png";
+import image3 from "./ProductImages/image3.png";
+import image4 from "./ProductImages/image4.png";
```

Comment:
Static images should be part of the public folder
https://stackoverflow.com/a/48730272